lib/watcher-middleware.js

```diff
@@ -34,6 +36,21 @@ module.exports = function watcherMiddleware(watcher, options) {
 
       next();
     }, (buildError) => {
+      if (request.url === '/ember_open_editor_for_template_compilation_error') {
+        let errorLocation = buildError.broccoliPayload.error.location || {};
+        let fileName = errorLocation.file;
+        let normalisedFileName = normaliseFileName(fileName);
+
+        launchEditor(normalisedFileName, () => {
+          if (!process.env.EDITOR || !process.env.VISUAL) {
```

Comment:
this condition contradicts with a console.log message:

    set the env variables EDITOR or VISUAL

which sounds like the condition should be changed to !process.env.EDITOR && !process.env.VISUAL

Reply:
Good catch..thanks :)